Dockerfile

```diff
@@ -1,5 +1,5 @@
 ############# builder
-FROM golang:1.15.3 AS builder
+FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
```

Comment:
Can you change also the alpine image below to `eu.gcr.io/gardener-project/3rd/alpine:3.12.1`? Just to be sure that we won't hit rate limit issue with the alpine image too.